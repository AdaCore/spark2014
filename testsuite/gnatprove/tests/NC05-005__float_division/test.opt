NT SKIP minor difference on windows
