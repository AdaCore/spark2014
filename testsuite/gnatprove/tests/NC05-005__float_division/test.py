from test_support import *

prove_all(prover=["alt-ergo","cvc4"])
