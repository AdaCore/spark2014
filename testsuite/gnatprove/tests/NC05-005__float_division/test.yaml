prove_all:
  prover: ["alt-ergo","cvc5"]
  steps: 5000
  counterexample: False
