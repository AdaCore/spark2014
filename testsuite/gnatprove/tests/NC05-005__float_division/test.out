pi_euler.adb:9:15: info: initialization of "Erreur" proved
pi_euler.adb:11:33: info: initialization of "Index" proved
pi_euler.adb:11:33: info: loop invariant initialization proved (altergo: 1 VC)
pi_euler.adb:11:33: info: loop invariant preservation proved (altergo: 1 VC)
pi_euler.adb:12:25: info: assertion proved (CVC4: 1 VC)
pi_euler.adb:12:36: info: initialization of "Index" proved
pi_euler.adb:12:36: info: range check proved (Interval)
pi_euler.adb:13:23: info: division check proved (CVC4: 1 VC)
pi_euler.adb:13:23: info: float overflow check proved (Interval)
pi_euler.adb:13:35: info: initialization of "Index" proved
pi_euler.adb:13:35: info: range check proved (Interval)
pi_euler.adb:14:16: info: initialization of "Pi" proved
pi_euler.adb:14:18: medium: float overflow check might fail
pi_euler.adb:14:19: info: initialization of "Erreur" proved
pi_euler.adb:15:19: info: initialization of "Index" proved
pi_euler.adb:15:24: medium: overflow check might fail (e.g. when Index = 0)
pi_euler.adb:19:12: info: initialization of "Pi" proved
pi_euler_2.adb:8:10: warning: unused assignment
pi_euler_2.adb:11:15: info: initialization of "Erreur" proved
pi_euler_2.adb:15:33: info: initialization of "Index_Float" proved
pi_euler_2.adb:15:33: info: loop invariant initialization proved (altergo: 1 VC)
pi_euler_2.adb:15:33: info: loop invariant preservation proved (CVC4: 1 VC)
pi_euler_2.adb:16:25: info: assertion proved (altergo: 1 VC)
pi_euler_2.adb:16:25: info: initialization of "Index_Float" proved
pi_euler_2.adb:17:23: info: division check proved (CVC4: 1 VC)
pi_euler_2.adb:17:23: info: float overflow check proved (Interval)
pi_euler_2.adb:17:24: info: initialization of "Index_Float" proved
pi_euler_2.adb:17:35: info: division check proved (CVC4: 1 VC)
pi_euler_2.adb:17:35: info: float overflow check proved (Interval)
pi_euler_2.adb:18:16: info: initialization of "Pi" proved
pi_euler_2.adb:18:18: medium: float overflow check might fail
pi_euler_2.adb:18:19: info: initialization of "Erreur" proved
pi_euler_2.adb:19:16: warning: unused assignment
pi_euler_2.adb:19:19: info: initialization of "Index" proved
pi_euler_2.adb:19:24: medium: overflow check might fail (e.g. when Index = 0)
pi_euler_2.adb:20:25: info: initialization of "Index_Float" proved
pi_euler_2.adb:20:37: info: range check proved (Interval)
pi_euler_2.adb:20:37: medium: float overflow check might fail
pi_euler_2.adb:24:12: info: initialization of "Pi" proved
